docs/process/process_areas/implementation/guidance/cpp/code_analysis.rst

```diff
@@ -0,0 +1,118 @@
+..
+   # *******************************************************************************
+   # Copyright (c) 2025 Contributors to the Eclipse Foundation
+   #
+   # See the NOTICE file(s) distributed with this work for additional
+   # information regarding copyright ownership.
+   #
+   # This program and the accompanying materials are made available under the
+   # terms of the Apache License Version 2.0 which is available at
+   # https://www.apache.org/licenses/LICENSE-2.0
+   #
+   # SPDX-License-Identifier: Apache-2.0
+   # *******************************************************************************
+
+Code Analysis C++
+#################
+
+.. gd_guidl:: Static Code Analysis
+   :id: gd_guidl__cpp__code_analysis
+   :status: valid
+   :complies: std_req__iso26262__software_1, std_req__iso26262__software_2, std_req__iso26262__software_3, std_req__iso26262__software_31
+
+   Guideline for Static Code Analysis
+
+Static Code Analysis
+====================
+In order to fulfil standard requirements a concept for *Static Code Analysis* needs to be established. Input for this is based upon *MISRA* and *ISO26262*.
```

Comment:
I'm not sure about the order/introduction here. Before talking about static code analysis, we need to talk about coding rules, or a "language subset".
MISRA defines such a language subset for C++.
Static code analysis tool support checking, that only that subset has indeed been used.

Reply:
Ah just found the second file. Some reference is missing.

--> Static code analysis tools support checking compliance to the :ref:codeing_guidelines.